fuzz/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "naga-fuzz"
```

Comment:
since this is local to the repository, does it make sense to call it the full "naga-fuzz", or could it be just "fuzz"?

Reply:
By default it will scaffold a project with the `<original crate>-fuzz` name. I presume it's done to avoid conflicts, in case there's some dependency also called `fuzz`.